src/plugins/workspace/public/components/workspace_form/types.ts

```diff
@@ -79,7 +79,7 @@ export interface WorkspaceFormProps {
   application: ApplicationStart;
   savedObjects: SavedObjectsStart;
   onSubmit?: (formData: WorkspaceFormSubmitData) => void;
-  defaultValues?: WorkspaceFormData;
+  defaultValues?: Partial<WorkspaceFormData>;
```

Comment:
The `WorkspaceFormProps` was been used in `WorkspaceDetailedForm`, shall we separate a new `WorkspaceDetailedFormProps` like below code?
```ts
export interface WorkspaceDetailedFormProps extends WorkspaceFormProps {
  defaultValues?: WorkspaceFormData;
}
```
The `defaultValues` in `WorkspaceFormProps` should be `Partial<WorkspaceFormSubmitData>`.